docs/admin/admission-controllers.md

```diff
@@ -142,6 +143,15 @@ applies a 0.1 CPU requirement to all Pods in the `default` namespace.
 
 See the [limitRange design doc](../design/admission_control_limit_range.md) and the [example of Limit Range](limitrange/) for more details.
 
+### InitialResources (experimental)
+
+This plug-in observes pod creation requests. For each container that does not specify compute resource requests or limits,
+the plug-in auto-populates a compute resource request based on historical usage of containers running the same image.
+If there is not enough data to make a decision the Request is left unchanged.
+When the plug-in sets a compute resource request, it annotates the pod with information on what compute resources it auto-populated.
```

Comment:
This document explains what Admission Control plugin is, specifies the list of available plugins, describes how to enable them and what is the recommended set of plugins. I think it's unnecessary to mention that InitialResources is enabled by default because it is not on the list of recommended plugins and to explain how enabled it because it is already done for any plugin.